templates/new.html.template

```diff
@@ -1,7 +1,7 @@
 {% extends "templates/base.html.template" %}
 {% block head %}
 
-    {% block title %}<title> New {{ name }}</title>{% endblock %}
+    {% block title %}<title> {{ name }}</title>{% endblock %}
```

Comment:
This removes "New" from all apps using `new.html.template`. A better solution is to implement the title block in the options subtemplate, which will take precedence over the title in this file.